routeupdate/hostgw/hostgw_windows.go

```diff
@@ -0,0 +1,98 @@
+package hostgw
```

Comment:
Needs: `//+build windows`?

Reply:
not really needed because of the `_windows` subfix.